app/assets/stylesheets/autocomplete.scss

```diff
@@ -27,3 +27,14 @@ ul.ui-autocomplete {
     }
   }
 }
+
+// this ensures that this helper is still available on screen readers: https://a11yproject.com/posts/how-to-hide-content/
```

Comment:
Love that you added a comment that also includes a URL for more info on the solution here 🤩